modules/installation-configuration-parameters.adoc

```diff
@@ -192,7 +192,12 @@ Only IPv4 addresses are supported.
 endif::agent,bare,ibm-power,ibm-z,vsphere[]
 
 ifdef::agent,bare,ibm-power,ibm-z,vsphere[]
+Consider the following information before you configure network parameters for your cluster:
+
 * If you use the {openshift-networking} OVN-Kubernetes network plugin, both IPv4 and IPv6 address families are supported.
+** If you want a host with a non-IPv6 network configuration to access a cluster with a configured IPv6 network, you can configure the cluster's firewall to block traffic for the IPv6 protocol.
```

Comment:
Is there more context for this? I'm a little confused why this would be necessary. If you have a v4 client talking to a dual stack cluster, it's always going to talk to the v4 endpoints because that's all it has access to.

If you don't want to allow v6 access to your cluster at all, you do have the option of not configuring v6 VIPs so the only way for external clients to talk to it is v4, but I'm not sure we need to call that out explicitly.

Reply:
Removed the line. 

> If you want a host with a non-IPv6 network configuration to access a cluster with a configured IPv6 network, you can configure the cluster's firewall to block traffic for the IPv6 protocol.